src/main/java/com/icthh/xm/ms/entity/service/metrics/MetricsAdapter.java

```diff
@@ -0,0 +1,42 @@
+package com.icthh.xm.ms.entity.service.metrics;
+
+public interface MetricsAdapter {
+    /**
+     * Increments the value into a counter. If counter is not exist, will be generated based on tags
+     *
+     * @param tags        List of tag`s pair for metrics (necessary to determine the required counter)
+     * @param amount      Value for inc - optional
+     * @param name        Counter name (necessary if counter not found) - optional
+     * @param description Description of counter (necessary if counter not found) - optional
+     */
+    void inc(String[] tags, Double amount, String name, String description);
```

Comment:
Suggest using full name `increment(...)`

Reply:
I wouldn't change. This is standard abbreviation

Reply:
+ when using this method, a verbose line is obtained

Reply:
do not see any reason to abbreviate word increment.
as you may note micrometer library does not do it as well.

Reply:
done